tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h

```diff
@@ -28,7 +28,8 @@
 #include <bluetooth/uuid.h>
 #include <bluetooth/gatt.h>
 
-#define WAIT_TIME (30 * 1e6) /*seconds*/
+#define WAIT_SECONDS 30                /* seconds */
+#define WAIT_TIME (WAIT_SECONDS * 1e6) /* microseconds*/
```

Comment:
Maybe use the existing `USEC_PER_SEC` macro instead?

Reply:
Changed